cob_obstacle_distance_moveit/src/cob_obstacle_distance_moveit/obstacle_distance_moveit.cpp

```diff
@@ -169,13 +160,14 @@ void ObstacleDistanceMoveit::calculateDistanceTimerCallback(const ros::TimerEven
         {
             std::string robot_self_name = selfcollision_it->first;
             collision_detection::AllowedCollision::Type type;
-            if(acm.getEntry(robot_link_name, robot_self_name, type))
+
+            if(acm_.getEntry(robot_link_name, robot_self_name, type))
             {
-                if(type == collision_detection::AllowedCollision::NEVER)
+                if(type == collision_detection::AllowedCollision::ALWAYS)
```

Comment:
hmm, I always had problems understanding the meaning of the entries in the SRDF
does `NEVER` here mean "collision never allowed" and `ALWAYS` here mean "collision always allowed"?

how does this related to the SRDF entry?
`<disable_collisions link1="arm_left_1_link" link2="arm_left_6_link" reason="Never" />`
(I read this like "do not check for collisions between `arm_left_1_link` and `arm_left_6_link` because they `Never` collide (it's kinematically not possible for them to collide thus this collision check can be saved because the result is known to be false=no_collision)
Is the `type` of the entry `arm_left_1_link` and `arm_left_6_link` then `collision_detection::AllowedCollision::NEVER` due to the line above?

In that case, I would assume that we would need to calculate distances only for pairs that are neither `NEVER` nor `ALWAYS` colliding, but do not appear in the SRDF...